analyses/molecular-subtyping-chordoma/01-Subtype-chordoma.Rmd

```diff
@@ -18,32 +18,42 @@ library(ggplot2)
 ### Read in data
 
 ```{r}
-data_dir <- file.path("..", "..", "data")
-histologies_df <- read_tsv(file.path(data_dir,
-                                     "pbta-histologies.tsv"),
-                           col_types = cols(
-                             molecular_subtype = col_character()
-                           ))
+# File path to subsetted files directory
+input_dir <- "chordoma-subset"
 ```
 
+Set up metadata
+
 ```{r}
-# TODO: update to use consensus file with annotations included in the data 
-# download?
-focal_cn_df <- 
-  read_tsv(file.path("..", "focal-cn-file-preparation", "results",
-                     "consensus_seg_annotated_cn_autosomes.tsv.gz"))
+# Read in non-subsetted metadata
+histologies_df <- readr::read_tsv(file.path("..", "..", "data", "pbta-histologies.tsv"))
+
+# Subset metadata
+subset_metadata <- histologies_df %>%
+  dplyr::filter(short_histology == "Chordoma") %>%
+  select(
```

Comment:
You're calling the `filter` function above using `dplyr::` but not doing the same for `select` here. Is this because `filter` was behaving similarly to `rename` as you noted in your [original comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/590#issue-382545384)? If so, then disregard this comment. If not, removing the `dplyr::` altogether (except for in the case of the `rename` function) or adding the `dplyr::` to the rest of the functions would be nice for consistency purposes.

Reply:
`filter` also has a base:: function that can mess with stuff sometimes.